package.json

```diff
@@ -104,7 +104,8 @@
   },
   "resolutions": {
     "@types/react": "17.0.66",
-    "@graasp/sdk": "1.6.0"
+    "@graasp/query-client": "1.10.0",
+    "immutable": "4.3.4"
```

Comment:
Nitpick: If you upgrade immutable on line 50 to 4.3.4 can you remove this resolution ? If not, then no worries.